BadHabbit/BadHabbit/Kit/Extentions/UILabelExtention.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  UILabelExtention.swift
```

Comment:
There is a nice thread I post into slack channel about naming files. Follow please convension

Reply:
Done